src/main/java/org/jenkinsci/plugins/workflow/steps/PwdStep.java

```diff
@@ -95,7 +97,13 @@ public boolean isTmp() {
 
         @Override protected String run() throws Exception {
             FilePath cwd = getContext().get(FilePath.class);
-            return (tmp ? WorkspaceList.tempDir(cwd) : cwd).getRemote();
+            Objects.requireNonNull(cwd);
+            if (tmp) {
+                cwd = WorkspaceList.tempDir(cwd);
+                if (cwd == null)
+                    throw new IOException("Failed to set up a temporary directory.");
```

Comment:
(also in #116)

Reply:
I know - but until that gets merged (and I merge in master here), not applying the same fix here caused spotbugs failures.

Reply:
Right, just noting that this is an accommodation to newer `jenkins.version` generally.